documentation/docs/strategies/cross-exchange-market-making.md

```diff
@@ -32,6 +32,16 @@ The end result: Hummingbot has sold the same asset at \$102 (❶) and purchased
 
 Initially, we assume that the maker exchange is an Ethereum-based decentralized exchange and that the taker exchange is Binance.
 
+### Adjusting Orders
+
+The maker price is calculated based on the price at which you can buy or sell on the taker exchange with a min_profitability added to it.
+
+For example if you can sell 1 ETH on taker exchange for 100 USDT on a volume weighted average price basis, then if min_profitability is 0.05, then maker price at which limit buy will be placed is 95 USDT.
+
+Now its possible that the top bid on the exchange is much lower, say 90 USDT. In this case if you enable adjust orders it will place at 90.01 (one price level above top bid).
```

Comment:
*it's*

what is a "price level"?

Reply:
https://www.grammarly.com/blog/its-vs-its/

Reply:
https://en.wikipedia.org/wiki/Order_book_(trading)#Price_levels

Reply:
"Now it is possible".

Reply:
we need to explain the term on our website.

Reply:
both are correct

Reply:
see google - 

<img width="1138" alt="Screen Shot 2019-09-09 at 4 57 12 PM" src="https://user-images.githubusercontent.com/26056161/64574100-effa0700-d322-11e9-8099-380fa707c398.png">

Reply:
dont mark stuff as resolved if its not.

Reply:
I thought it was resolved. Ok, you want to be a grammar nazi I guess.

Reply:
Grammar police